project-base/src/Shopsys/ShopBundle/Controller/Front/CartController.php

```diff
@@ -2,9 +2,8 @@
 
 namespace Shopsys\ShopBundle\Controller\Front;
 
-use Shopsys\FrameworkBundle\Component\Controller\ErrorService;
-use Shopsys\FrameworkBundle\Component\Controller\FrontBaseController;
 use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Component\Controller\ErrorService;
```

Comment:
Im not sure how to fix this, because this is solved in other commit where ErrorService gets moved to FlashMessage Component folder. I could jump into rebase interactive and solve it, but it feels unnecessary to me. What do you think, do you really request this change?

Reply:
It's ok :slightly_smiling_face: